public/index.html

```diff
@@ -27,7 +30,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>EOSCR Web App Boilerplate</title>
+    <title>Notario Digital</title>
```

Comment:
maybe leave this as an env var?

Reply:
I changed that text without reason hehehe
In fact, when you send me the titles per page the component will change it automatically, we don't need an env var